src/main/kotlin/org/malachite/estella/task/api/TaskExceptionHandler.kt

```diff
@@ -0,0 +1,21 @@
+package org.malachite.estella.task.api
+
+import org.malachite.estella.commons.Message
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.task.domain.InvalidTestFileException
+import org.malachite.estella.task.domain.TaskNotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
+class TaskExceptionHandler {
+    @ExceptionHandler(TaskNotFoundException::class)
+    fun handleTaskNotFound(ex: TaskNotFoundException): ResponseEntity<Message> =
+        OwnResponses.NO_RESOURCE("This task doesn't exist or you don't have access to it")
```

Comment:
Jeśli dostaniesz to na froncie, to się front wywali, jako że powinieneś wtedy robić `response.text()`, a wszędzie robimy `response.json()`. Także lepiej to opakować w `Message`, chyba że jest to robione w OwnResponses.

Reply:
Jest wrapowane w Message pod spodem